advanced-menu/advanced-menu.py

```diff
@@ -51,11 +52,14 @@ def __init__(self, bot, msg, thread, config: dict, data: dict, is_home: bool):
     async def callback(self, interaction: discord.Interaction):
         try:
             # await interaction.response.send_message("You selected {}".format(self.values[0]))
+            selectionType = "submenu" if self.values[0] == "Main menu" else self.data[self.values[0]]["type"]
```

Comment:
What is the point of rewriting this? It's also overly complex.

Reply:
I rewrote this so it could be used in the method call below, but stored it as a variable since I needed that exact expression for the method call but it could also be used in the else if statement. If you have a better way to simplify it, let me know, but the only change from before is that Main menu is classified as a submenu to fix a bug.